wiki/quest-modding.md

```diff
@@ -106,6 +106,59 @@ Continue to the [Core Mods](#core-mods) step of the installation process.
 * A Cable to connect your Quest to your Phone (If your phone charges over USB C the charger that was included with your
 Quest should work)
 
+
```

Comment:
This is chonky and should be redone in some way or put in an entirely seperate file

Reply:
Considering the other suggestion adding more chonk, you are welcome to creating a separate md file named `quest-modding-with-phone.md` under the support folder

Reply:
Should be there now with a link on the normal modding guide. You might want to check the links as I'm not sure if they're right.